src/workerd/util/sqlite.c++

```diff
@@ -387,7 +387,7 @@ kj::Own<sqlite3_stmt> SqliteDatabase::prepareSql(
     SQLITE_REQUIRE(result != nullptr, "SQL code did not contain a statement.", sqlCode);
     auto ownResult = ownSqlite(result);
 
-    while (*tail == ' ' || *tail == '\n') ++tail;
```

Comment:
Just a side question here: should this also account for other kinds of whitespace? tabs, non-breaking spaces, etc (https://jkorpela.fi/chars/spaces.html)

Reply:
It looks like SQLite's own behaviour for whitespace (i.e escaping in the shell when using `--backslash` or their `isspace` functions) also cover `\t`, `\v` and `\f` in addition to ` `, `\r` and `\n`.

https://github.com/sqlite/sqlite/blob/11223e0cff820fc81ebe6cf00192aec4bd7be521/src/shell.c.in#L5668-L5685

https://github.com/sqlite/sqlite/blob/11223e0cff820fc81ebe6cf00192aec4bd7be521/ext/fts3/fts3_expr.c#L116-L118

Reply:
If we added the rest of the whitespace characters that SQLite covers, it might make sense to move to comparing `*tail` to hex ranges (good catch @DaniFoldi).

```cpp
while (*tail == ' ' || *tail == '\r' || *tail == '\n' || *tail == '\f' || *tail == '\t' || *tail == '\v' ) ++tail;

while (*tail == 0x20 || (*tail >= 0x09 && *tail <= 0x0d)) ++tail;
```

It doesn't overrun on a single line but could be a little confusing as to what it's doing at a glance, but a simple comment should solve that.